src/pushover_complete/pushover_api.py

```diff
@@ -173,7 +183,7 @@ def send_message(self, user, message, device=None, title=None, url=None, url_tit
         :returns: Response body interpreted as JSON
         :rtype: dict
         """
-        return self._send_message(user, message, device, title, url, url_title, priority, retry, expire, callback_url,
+        return self._send_message(user, message, device, title, url, url_title, image_path, priority, retry, expire, callback_url,
```

Comment:
not specifically aimed at your PR, but I'd rather see kwargs used in this long call. it would make reading diffs easier too!

Reply:
True, I briefly considered that, but didn't want to complicate the PR. I'd suggest a general issue tracking that across the whole project.

Reply:
@scolby33 is going to go crazy and add type annotations so its 36+ only next time he does this, so be careful what you wish for :p

Reply:
I'm still unsure on this. It seems to make discovery of the possible parameters more difficult...but I do admit that this signature is atrocious.